README.md

```diff
@@ -68,6 +68,8 @@ var readyEvent = new CustomEvent("rise-component-ready", {
 this.dispatchEvent(readyEvent);
 ```
 
+The component has 30 seconds in which to initialize and dispatch the `rise-component-ready` event. After that, only components that have reported as ready will display. Components that are skipped will have their playlist item duration (if specified) added to the duration of the next playlist item following it that reported as being ready.
```

Comment:
@donnapep The last sentence *"Components that are skipped will have their playlist item duration (if specified) added to the duration of the next playlist item following it that reported as being ready."* doesn't read well at the end. I think it's the use of "that". Can you adjust?